rs/xnet/payload_builder/src/certified_slice_pool.rs

```diff
@@ -352,14 +355,12 @@ struct Payload {
 }
 
 /// Mean empty payload byte size: `LabelTree` with `"streams"` and `"header"`
-/// labels; subnet ID; and serialized header (3 variable length encoded
-/// integers).
-const EMPTY_PAYLOAD_BYTES: usize = 62;
+/// labels; subnet ID.
```

Comment:
Nit: I would explicitly mention that this leaves out the actual header blob.

```suggestion
/// Mean empty payload byte size: `LabelTree` with subnet ID, `"streams"` and
/// `"header"`labels; but excluding the header leaf.
```

Reply:
Changed.